### PR TITLE
Ruby

### PR DESCRIPTION
**Pull request by JohnForster**
Plumbing done to connect ruby to server. Also refactoring within script and preventing duplicate file for testing.

### CODE REVIEW

File: server/src/script.test.js [hunk 1]
```diff
@@ -2,51 +2,79 @@
...
-  });
+    it('returns .toString when appropriate', () => {
+      expect(findMethod([1, 2, 3], '1,2,3')).toContain('toString');
+    });
 
-  it('returns .toString when appropriate', () => {
-    expect(findMethod([1, 2, 3], '1,2,3')).toContain('.toString');
-  });
+    describe ('with nested arrays', () => {
+      it('returns .pop for a nested array', () => {
```
Comment (elishkaflint) [inline server/src/script.test.js:31]: please can we add feature tests for ruby as we have for javascript?

**Review by elishkaflint — APPROVED** [review]
:ant:

File: server/src/Gemfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
```
Comment (bibicollins) [inline server/src/Gemfile]: Do we need a second gemfile?

**Review by bibicollins — APPROVED** [review]
:ant:

**Pull request merged**